docs/zkapps/roadmap.mdx

```diff
@@ -83,7 +83,7 @@ To view all commands offered by the zkApp CLI, install it via `npm install -g zk
 
 ### Other
 
-- Archive Node API — Phase I; link to RFC on Github coming; ETA Q1 2023
+- Archive Node API — Phase I; ETA Q1 2023
```

Comment:
removed the "coming soon" since the best docs guidelines are to omit any references to future actions, per https://developers.google.com/style/future